File: lab3/s1/ejb/src/main/java/ru/uvuv643/ejb/remote/dto/human/CarDto.java
package ru.uvuv643.ejb.remote.dto.human;

import jakarta.validation.Valid;
import  jakarta.xml.bind.annotation.XmlAccessType;
import  jakarta.xml.bind.annotation.XmlAccessorType;
import  jakarta.xml.bind.annotation.XmlAttribute;
import  jakarta.xml.bind.annotation.XmlRootElement;
import java.io.Serializable;

@XmlRootElement(name = "CarDto")
@XmlAccessorType(XmlAccessType.FIELD)
public class CarDto implements Serializable {

        @Valid
        @XmlAttribute(name = "cool")
        private Boolean cool;

        public CarDto() {
                this(null);
        }

        public CarDto(Boolean cool) {
                this.cool = cool;
        }

        public Boolean getCool() {
                return cool;
        }

        public void setCool(Boolean cool) {
                this.cool = cool;
        }

        @Override
        public String toString() {
                return "Car {" +
                        "cool='" + cool + "'}";
        }

}

File: lab2/s1/src/main/kotlin/ru/uvuv643/soa/api/v1/dto/human/CarDto.kt
package ru.uvuv643.soa.api.v1.dto.human

import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty
import io.swagger.v3.oas.annotations.media.Schema
import jakarta.annotation.Nullable
import jakarta.validation.Valid
import jakarta.xml.bind.annotation.*

@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
data class CarDto constructor(

        @field:Valid
        @field:Nullable
        @XmlAttribute(name = "cool")
        val cool: Boolean? = null,

        )